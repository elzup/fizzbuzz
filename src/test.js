// @flow
const M = require('.').default

test('snapshot', () => {
  const m = new M()
  expect(m.take()).toMatchSnapshot()

  const t = m.it()
  expect(t.next().value).toMatchSnapshot()
  expect(t.next().value).toMatchSnapshot()
  expect(t.next().value).toMatchSnapshot()
})
