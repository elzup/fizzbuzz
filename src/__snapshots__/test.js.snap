// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it: works 1`] = `
Array [
  "1",
  "2",
  "Fizz",
  "4",
  "Buzz",
]
`;

exports[`take: to 3 1`] = `
Array [
  "1",
  "2",
  "Fizz",
]
`;

exports[`take: to 10 from 5 1`] = `
Array [
  "Buzz",
  "Fizz",
  "7",
  "8",
  "Fizz",
  "Buzz",
]
`;

exports[`take: to 30 1`] = `
Array [
  "1",
  "2",
  "Fizz",
  "4",
  "Buzz",
  "Fizz",
  "7",
  "8",
  "Fizz",
  "Buzz",
  "11",
  "Fizz",
  "13",
  "14",
  "FizzBuzz",
  "16",
  "17",
  "Fizz",
  "19",
  "Buzz",
  "Fizz",
  "22",
  "23",
  "Fizz",
  "Buzz",
  "26",
  "Fizz",
  "28",
  "29",
  "FizzBuzz",
]
`;
